sys/include/vfs_util.h

```diff
@@ -115,6 +115,15 @@ int vfs_file_sha256(const char* file, void *digest,
  */
 int vfs_is_dir(const char *path);
 
+/**
+ @brief Checks if @p path is a file and can be read.
+ *
+ * @param[in]   path        Path to check
+ *
+ * @return true if the file exists, false otherwise
+ */
+bool vfs_file_exists(const char *path);
```

Comment:
`#include <stdbool.h>` is missing and when you are at it, also add missing `stddef.h` for `size_t`.